test/address.spec.ts

```diff
@@ -556,39 +586,14 @@ describe('address', () => {
 
       describe('nearbyGPSCoordinate()', () => {
         it('should return random gps coordinate within a distance of another one', () => {
-          function haversine(lat1, lon1, lat2, lon2, isMetric) {
-            function degreesToRadians(degrees) {
-              return degrees * (Math.PI / 180.0);
-            }
-            function kilometersToMiles(miles) {
-              return miles * 0.621371;
-            }
-            const R = 6378.137;
-            const dLat = degreesToRadians(lat2 - lat1);
-            const dLon = degreesToRadians(lon2 - lon1);
-            const a =
-              Math.sin(dLat / 2) * Math.sin(dLat / 2) +
-              Math.cos(degreesToRadians(lat1)) *
-                Math.cos(degreesToRadians(lat2)) *
-                Math.sin(dLon / 2) *
-                Math.sin(dLon / 2);
-            const distance = R * 2 * Math.atan2(Math.sqrt(a), Math.sqrt(1 - a));
-
-            return isMetric ? distance : kilometersToMiles(distance);
-          }
-
-          let latFloat1: number;
-          let lonFloat1: number;
-          let isMetric: boolean;
-
-          for (let i = 0; i < 10000; i++) {
-            latFloat1 = parseFloat(faker.address.latitude());
-            lonFloat1 = parseFloat(faker.address.longitude());
-            const radius = Math.random() * 99 + 1; // range of [1, 100)
-            isMetric = Math.round(Math.random()) === 1;
+          for (let i = 0; i < 100; i++) {
```

Comment:
At least thousends, maybe even back to 10_000?

```suggestion
          for (let i = 0; i < 1_000; i++) {
```

```suggestion
          for (let i = 0; i < 10_000; i++) {
```

Reply:
I wouldn't recommend that, as this just wastes time. We don't do the NON_SEEDED_BASE_RUN that often either.